shared/src/main/scala-3.0.0-RC2/urldsl/vocabulary/FromStringWithNumeric.scala

```diff
@@ -0,0 +1,18 @@
+package urldsl.vocabulary
+
+import urldsl.errors.ErrorFromThrowable
+
+trait FromStringWithNumeric {
```

Comment:
I'm not sure what this should be for Scala 3. This trait is empty in 2.12, I copied this from 2.13